fideslib/oauth/api/scope_registry.py

```diff
@@ -0,0 +1,94 @@
+CLIENT_CREATE = "client:create"
```

Comment:
Several of these already exist, with additional documentation that improves the auto-generated docs, here: https://github.com/ethyca/fideslib/blob/main/fideslib/oauth/scopes.py

I'd rather we continue that pattern to ensure our scopes are well-documented.

Reply:
@seanpreston does it impact fidesops if I make this change? What I have currently is a direct copy from fidesops.

or @PSalant726  do you just mean adding the "top level" and swapping the `"client:create"` parts with `f"{CLIENT}:{CREATE}"`:

```py
CREATE = "create"
DELETE = "delete"
READ = "read"
UPDATE = "update"
CLIENT = "client"
```

Reply:
The use of f-strings is less important than the existence of the `SCOPE_DOCS` dict, and the population of its values. It's referenced [here](https://github.com/ethyca/fideslib/blob/main/fideslib/oauth/api/utils.py#L12), and enhances the auto-generated documentation.

Reply:
OK can we consolidate these two files into one then, scope_registry and scopes, keeping the `SCOPE_DOCS` dict?

Reply:
I merged these files and made a guess at the text for the docs, but I won't be surprised if some of those need updating.